presto-docs/src/main/sphinx/release/release-0.238.rst

```diff
@@ -0,0 +1,32 @@
+=============
+Release 0.238
+=============
+
+General Changes
+_______________
+* Fix case sensitivity issue in SQL function parameters. Function parameters should be treated case-insensitive.
+* Add ``target_result_size`` session property to customize data batch sizes being streamed from coordinator.
+* Add optimization to push null filters to the INNER side of equijoins. The optimization can be enabled with `optimize-nulls-in-joins`.
```

Comment:
Use pairs of double back quotes instead of single back quotes for proper formatting. Here and below. https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#formatting